test/nd_list_test.dart

```diff
@@ -140,7 +187,7 @@ void main() {
     });
   });
 
-  group('NDList<double> indexing', () {
+  group('Basic operators', () {
     test('==', () {
```

Comment:
That's correct, though the preferred method is `expect(ndList, equals(ndList))` since then Dart will explain that the two are not equal, and why, in the test failure. (Versus "This statement is not true." as you get from the other.)